### DISCUSSION TITLE
Is this library the same as the one mentioned but missing on IEEE docs website?

### DISCUSSION THREAD

**Discussion by SRM256** [GridProtectionAlliance/gsf#80] [Q&A] [answered]
The IEEE documents website for the PQDIF spec:
[https://grouper.ieee.org/groups/1159/3/docs.html](https://grouper.ieee.org/groups/1159/3/docs.html)

On that page, there is a subject link for "PQDIF .NET Library" and under that is a secured link for the source code of the library, but (unlike the C library), the DLL itself is missing. Before I found GSF, I contacted the IEEE owners listed on the page but got no response.

I am wondering: Is the library mentioned there this library? Or is GSF is an independent creation?
_(This has no particular bearing on your library, I'm just curious and trying to figure out if there some other library that is the standard definition library that maybe I should be recommending that my company use.)_

**Comment by StephenCWills** [comment, marked as answer]
GSF.PQDIF is an independent creation based almost entirely on the text in the IEEE P1159.3 Draft 9 standards document from 2002. I've actually never seen the source code of PQDIFNET.DLL or tried to use it so I can't really provide any useful input beyond that. Hopefully they'll send you a response soon so you can give it a fair evaluation.
